pkg/scheduler/api/job_info.go

```diff
@@ -104,7 +104,7 @@ func getJobID(pod *v1.Pod) JobID {
 
 // NewTaskInfo creates new taskInfo object for a Pod
 func NewTaskInfo(pod *v1.Pod) *TaskInfo {
-	req := GetPodResourceWithoutInitContainers(pod)
+	req := GetPodResourceRequest(pod)
```

Comment:
The relate issue is https://github.com/volcano-sh/volcano/issues/1333, which makes clear analysis about the the root reason. We should get the max(<init container resources>), sum(<containers in pod>).

Reply:
Well, this fix is right.

Reply:
/lgtm